src/cmds/qstat.c

```diff
@@ -89,7 +90,7 @@ char *cnvt_est_start_time(char *start_time, int shortform);
 #define ALT_DISPLAY_Mb	0x100	/* show sizes in MB */
 #define ALT_DISPLAY_Mw	0x200	/* -M option - show sizes in MW */
 #define ALT_DISPLAY_G	0x400	/* -G option - show sizes in GB */
-#define ALT_DISPLAY_1l  0x800	/* -n -s on line line */
+#define ALT_DISPLAY_1l  0x800	/* -n -s -f on line line */
```

Comment:
Hi @dilip-krishnan 
We will continue this discussion in the community forum as that will give an opportunity for those who have already engaged in the EDD discussion. Cheers :)